### PR TITLE
Create brand_impersonation_hulu.yml

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

**Comment by zoomequipd** [comment]
this never landed in test rules for some reason.

**Comment by aidenmitchell** [comment]
Maybe? Just `hulu` feels pretty loose.

**Comment by aidenmitchell** [comment]
No hits

File: detection-rules/brand_impersonation_hulu.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      and any(ml.logo_detect(beta.message_screenshot()).brands,
+              .name == "Hulu"
+              and .confidence in ("medium", "high")
+      )
+    )
+  )
+  and sender.email.domain.root_domain not in (
+    'hulu.com'
+  )
+  and sender.email.email not in $recipient_emails
```
Comment (zoomequipd) [inline detection-rules/brand_impersonation_hulu.yml:28]: solicited? 
```suggestion
  and not profile.by_sender().solicited
```

**Review by zoomequipd — APPROVED** [review]

**Pull request merged**